demos/features/sync.mesh.html

```diff
@@ -41,28 +41,29 @@ <h1>
     </section>
     <script src="./niivue.umd.js">
     </script>
-    <script>
+    <script type="module" async>
       document.getElementById("check10").addEventListener("change", doCheck10Click);
       function doCheck10Click() {
         nv1.setHighResolutionCapable(this.checked);
       }
       var nv1 = new niivue.Niivue({backColor: [1, 1, 1, 1]})
       nv1.attachTo('gl1')
       nv1.setHighResolutionCapable(false)
-      nv1.setMeshShader('Outline')
       nv1.opts.isOrientCube = true
-      nv1.loadMeshes([
+      await nv1.loadMeshes([
          {url: "../images/BrainMesh_ICBM152.lh.mz3"},
       ])
```

Comment:
We can prob do something like this:

```
nv1.loadMeshes([
  {url: "../images/BrainMesh_ICBM152.lh.mz3"},
]).
then((id)=>{
nv1.setMeshShader(id, 'Outline')
}
```

Reply:
@hanayik sounds fine to me. Do you want to implement this with the relevant /demos/features/*.html files.